modules/las/src/lib/laslaz-decoder.ts

```diff
@@ -323,6 +323,7 @@ class LAZLoader {
   close(): boolean {
     try {
       if (this.instance !== null) {
+        Module._free(this.instance.buf);
```

Comment:
Appreciate the PR. I am not familiar enough with the underlying code to be able to assess this change. I do notice that we call `instance.delete()` and I would expect that necessary cleanup is performed inside that method. If we are calling it outside, perhaps at least add a comment explaining that there is a resource leak in instance.delete() and that we can't fix that code.

Reply:
Forgot to mention this.

https://github.com/visgl/loaders.gl/blob/314af6c3a405d2d71216f588a43ec974c891d6cf/modules/las/src/lib/laslaz-decoder.ts#L237-L245  
The `buf` is not managed by `instance`. It's allocated in the `open` method.   
So there is no resource leak in instance.delete().